python/dask_cudf/dask_cudf/tests/test_binops.py

```diff
@@ -72,3 +72,22 @@ def test_series_binops_float(binop):
     got = binop(lhs_gdf.x, rhs_gdf.y)
     exp = binop(lhs_df.x, rhs_df.y)
     dd.assert_eq(got, exp)
+
+
+@pytest.mark.parametrize("operator", _binops)
+def test_df_series_bind_ops(operator):
```

Comment:
This doesn't test the new operator methods does it (e.g pow)?

Reply:
I think what gets called is `getattr(lhs_gdf[col], operator.__name__)` which points to `<bound method Series._bind_operator_method.<locals>.meth of <dask_cudf.Series | 4 tasks | 2 npartitions>>`, that's what I want right?

Reply:
Yes - I do think you want to test `_bind_operator_method`,  but won't `operator` only correspond to [these specific binary ops](https://github.com/rapidsai/cudf/blob/091287a6e0c638d7703f7ef575ae29eac31048b4/python/dask_cudf/dask_cudf/tests/test_binops.py#L40) in this case?

Reply:
I guess I'm suggesting that the test is fine, but it may make sense to cover something closer to the reproducer in the original issue (however, maybe this is unnecessary).  What do you think?

Reply:
oh whoops I didn't notice `mod` and `pow` weren't actually in the list, should be fixed.